web/index.html

```diff
@@ -1463,19 +1474,24 @@ <h4 class="modal-title" id="switchRegistryModalLabel">Switch Netdata Registry Id
 	},
 
 	'disk.iotime': {
-		height: 0.5
+		height: 0.5,
+		info: 'Number of milliseconds spend by all operation. Read from fields 4 and 8.'
 	},
 	'disk.mops': {
-		height: 0.5
+		height: 0.5,
+		info: 'Number of merged operations. Read from fields 2 and 6.'
```

Comment:
The number of merged disk operations. The Linux kernel is able to merge adjacent I/O operations, for example two 4KB reads can become one 8KB read before it is handled to the disk.